bin/test.py

```diff
@@ -57,8 +65,24 @@ def get_log_dirs(path: Path):
 logger = TensorBoardLogger(
     save_dir=log_dir, name=folder, version=subfolder, default_hp_metric=False
 )
+run_id = datetime.now().strftime("%b-%d-%Y-%M-%S")
 
-trainer = pl.Trainer(logger=logger, devices="auto", fast_dev_run=args.only_plots)
+# Setup profiler
+if args.profiler == "pytorch":
```

Comment:
I see this is the same in `bin/train.py`, but it seems impossible to use the "simple" profiler, while it is "pytorch" or nothing. Maybe "simple" could be removed from the `--profiler` possibilities ?

Reply:
Oops to slow to write the comment :)